spec/javascripts/Policies/actions/PolicyRegistry.spec.jsx

```diff
@@ -1,6 +1,6 @@
 import { fetchRegistry } from 'Policies/actions/PolicyRegistry'
 
-describe('Policy Registry Actions', () => {
+describe('Policies Registry Actions', () => {
```

Comment:
_PolicyRegistry_ actions would be more accurate, since the component it refers to the component.

Reply:
I don't get what you mean by

>the component it refers to the component.

"Policies Registry" was an attempt to make it clear it is about a registry of possible multiple custom policies, as opposed to a single policy registry. Even though "Policies Registry" is not grammatically incorrect AFAIK, it's probably a good thing to keep it "Policy Registry" as you said for consistency with other (most) API endpoints that also describe actions on collections of records.

Currently most of the API endpoints are in the form "Item Collection" (singular-singular), such as "Invoice List" instead of "Invoices List". Although we do have exceptions already, like "Account Features List" (plural-singular). The current endpoint to fetch APIcast policies spells it "APIcast Policies Registry" (plural-singular). I think I'll have to change that too.

Reply:
Should we also change the name of the rolling update and the name of the member permission section?

cc @josemigallas @hallelujah @secretsurf @thomasmaas

Reply:
It feels that we might want to refactor at some point to keep consistency.

Reply:
Changed everything to "policy registry", including the title of the already existing API endpoint 
"APIcast Policies Registry" (now "APIcast Policy Registry")